community/bolt/transport-socket/pom.xml

```diff
@@ -50,6 +50,15 @@
     </dependency>
 
     <dependency>
+      <groupId>org.bouncycastle</groupId>
+      <artifactId>bcprov-jdk15on</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.bouncycastle</groupId>
+      <artifactId>bcpkix-jdk15on</artifactId>
```

Comment:
Bolt tests are still executed during the build. And without bouncycastle some tests fail with because neither `sun.security.x509` nor `org.bouncycastle.cert` are in the classpath.
If Bolt is going to be removed than it is better to wait with this PR and change only server pom.